text/0000-channel-participation-api-without-system-channel.md

```diff
@@ -0,0 +1,497 @@
+
+## Privacy problems
+A system channel requires that all ordering service nodes (OSNs) be aware of all channels. As Fabric depricated the 
+Kafka-based ordering service and moved to Raft (and later is planned to introduce BFT consensus), it is no longer 
+desirable that an OSN that services a channel `A` with member organizations `X & Y` be aware of channel `B` with member 
+organizations `Y & Z`. This creates a privacy problem, because the orderer organization (or organizations) now knows 
+that `Y`, `X`'s partner to channel `A`, is also doing business with organization `Z`.  
+
+There are workarounds that reduce the privacy problem, but not eliminate it. For example, using the system channel it 
+is possible to create an application channel with only a single org, and then add additional orgs by updating the 
+channel itself. However, these type of solutions are cumbersome and only partial.
+
+## Scalability problems
+All OSNs are members of the system channel, which creates a scalability problem in a large scale Fabric network. When 
+the number of channels increases, Raft allows us do decouple the consenters sets of the different application channels, 
+achieving linear horizontal scalability in the number of channels. That is, more resource can be added as the number of 
+channels increase. However, the system channel is an exception: decoupling application channel consenters sets as 
+described above will cause its number of members to increase, and hence its performance to decrease. Joining and 
+leaving channels without a system channel solves this problem.     
+   
+Moreover, in order to start a new orderer and have it join an existing channel (perform 'on-boarding'), a new orderer 
+starts by scanning the system channel in order to discover existing channels and their membership. As the number of 
+channels increase, this scan prolongs the process of adding a new orderer.  
+    
+## Operational problems
+There are several operational problems that affect large scale deployment and hinder efficient management of cloud 
+environments using automatic tooling.
+     
+Channel creation (using the system channel) utilizes information contained in the system channel - e.g. MSP information 
+of the organization - in order to apply it onto the newly created channel. This create the convoluted situation in 
+which an organization that wants to update its details in the consortium must coordinate that action with third parties 
+that have write privileges to the system channel.
+
+There is no good way to list all the channels that an orderer is a member of. 
+
+There is no good way to dispose of the resources associated with a channel, once an orderer is no longer a member of 
+the consenters set of a that given channel.
+
+In many cloud environments, it is necessary to restart an orderer in order for it to discover immediately that it has 
+been added to a channel it was not a part of previously, because otherwise detection takes minutes. 
```

Comment:
These should probably be a bulleted list for better readability

Reply:
Done